homeassistant/components/configurator.py

```diff
@@ -105,14 +133,16 @@ def __init__(self, hass):
         self._cur_id = 0
         self._requests = {}
         hass.services.async_register(
-            DOMAIN, SERVICE_CONFIGURE, self.handle_service_call)
+            DOMAIN, SERVICE_CONFIGURE, self.async_handle_service_call)
 
-    def request_config(
+    @async_callback
+    def async_request_config(
```

Comment:
Everything that is in the init method would be moved to the async_get_instance callback